packages/bp/src/core/messaging/messaging-client.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios'
+
+export class MessagingClient {
+  private apiUrl: string
+
+  constructor(
+    public baseUrl: string,
+    private password: string,
+    private clientId?: string,
+    private clientToken?: string
+  ) {
+    this.apiUrl = `${this.baseUrl}/api`
+  }
+
+  async syncClient(config: any) {
```

Comment:
Please avoid any. Shouldn't it be MessagingConfig ?

Reply:
This code gets replaced in PR 9 with the botpress/messaging-client package